frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -401,6 +444,45 @@ describe("collection permissions", () => {
               });
             });
 
+            describe("managing question from the question's edit dropdown", () => {
+              it("should not be offered to save question in collections they have `read` access to", () => {
+                cy.visit("/question/1");
+                cy.icon("pencil").click();
+                popover()
+                  .findByText("Add to dashboard")
+                  .click();
+
+                cy.get(".Modal").within(() => {
+                  cy.findByText("Orders in a dashboard").should("not.exist");
+                  cy.icon("search").click();
+                  cy.findByPlaceholderText("Search").type(
+                    "Orders in a dashboard{Enter}",
+                  );
+                  cy.findByText("Orders in a dashboard").should("not.exist");
+                });
+              });
+
+              it("should be offered to save question in personal collection when creating a new dashboard", () => {
+                const { first_name, last_name } = USERS[user];
+                const personalCollection = `${first_name} ${last_name}'s Personal Collection`;
+                cy.visit("/question/1");
+                cy.icon("pencil").click();
+                popover()
+                  .findByText("Add to dashboard")
+                  .click();
+
+                cy.get(".Modal").within(() => {
+                  cy.findByText("Create a new dashboard").click();
+                  cy.get(".AdminSelect").findByText(personalCollection);
+                  cy.findByLabelText("Name").type("Foo");
+                  cy.findByRole("button", { name: "Create" }).click();
+                });
+                cy.url().should("match", /\/dashboard\/\d+$/);
+                saveDashboard();
+                cy.get(".Entity").findByText(personalCollection);
```

Comment:
Nitpick: `. DashboardHeader` conveys more meaning. I had no idea whatsoever what is `.Entity` so I had to build the test and observe what's going on in the Cypress runner.

Don't have to change it in this PR. Just a general note.

Reply:
Fixed, it's much better now. Thank you 🙏